prow/jobs/kyma/skr-kyma-to-kyma2-upgrade-aws.yaml

```diff
@@ -42,20 +43,16 @@ periodics: # runs on schedule
             securityContext:
               privileged: true
             command:
-              - "make"
-            args:
-              - "-C"
-              - "/home/prow/go/src/github.com/kyma-project/kyma/tests/fast-integration"
-              - "ci-skr-kyma-to-kyma2-upgrade"
+              - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-upgrade-skr-aws-kyma-to-kyma2.sh"
             env:
               - name: KEB_PLAN_ID
                 value: "361c511f-f939-4621-b228-d0fb79a1fe15"
               - name: KYMA_MAJOR_VERSION
                 value: "1"
-              - name: KYMA_UPGRADE_VERSION
-                value: "2.0.3"
+              - name: KYMA_PROJECT_DIR
```

Comment:
Do you manually add this value or it's generated from the template file `skr-kyma-to-kyma2-upgrade-aws-data.yaml`? I haven't find `KYMA_PROJECT_DIR` in that file.